06_gpu_and_ml/dreambooth/dreambooth_app.py

```diff
@@ -516,14 +518,14 @@ async def background():
 #
 # - `modal run dreambooth_app.py` will train the model. Change the `instance_example_urls_file` to point to your own pet's images.
 # - `modal serve dreambooth_app.py` will [serve](https://modal.com/docs/guide/webhooks#developing-with-modal-serve) the Gradio interface at a temporary location. Great for iterating on code!
-# - `modal shell dreambooth_app.py` is a convenient helper to open a bash [shell](https://modal.com/docs/guide/developing-debugging#stubinteractive_shell) in our image. Great for debugging environment issues.
+# - `modal shell dreambooth_app.py` is a convenient helper to open a bash [shell](https://modal.com/docs/guide/developing-debugging#appinteractive_shell) in our image. Great for debugging environment issues.
```

Comment:
Is this going to work?

Reply:
i think it's actually already broken – the link should be https://modal.com/docs/guide/developing-debugging#interactive_shell

will update, thanks!